server/configuration.js

```diff
@@ -20,6 +22,11 @@ export const setEnvironment = () => {
             process.env.TRAY_ENDPOINT = productionGraphqlEndpoint;
             process.env.TRAY_APP_URL = appUrlProd;
             break;
+        case "eu1-prod":
```

Comment:
Can the README be updated to include the new eu1 parameter? There are several instances in that document where it explicitly lists the current environments as "prod / fe-stag / stg"
I believe they're all in: 
https://github.com/trayio/embedded-edition-sample-app/blob/master/README.md#setting-up-and-running-the-sample-application